R/utilities-parameters.R

```diff
@@ -67,10 +67,19 @@ exportParametersToXLS <- function(parameters, paramsXLSpath, sheet = NULL) {
 
   for (paramIdx in seq_along(parameters)) {
     param <- parameters[[paramIdx]]
-    parameterContainerPath[[paramIdx]] <- param$parentContainer$path
-    parameterName[[paramIdx]] <- param$name
-    parameterUnits[[paramIdx]] <- param$unit
-    parameterValue[[paramIdx]] <- param$value
+    value <- param$value
+    if (!is.nan(value)){
+      parameterContainerPath[[paramIdx]] <- param$parentContainer$path
+      parameterName[[paramIdx]] <- param$name
+      parameterUnits[[paramIdx]] <- param$unit
+      parameterValue[[paramIdx]] <- param$value
+    } else {
```

Comment:
With this implementation, the function creates an excel file with empty rows. Is there a good way to remove all `NAs` before exporting?

Reply:
@PavelBal  Should be. 

Which unit test / reproducible example I can use to work on this ?

Reply:
```
sim <- loadSimulation(system.file("extdata", "Aciclovir.pkml", package = "ospsuite"))

param1 <- getParameter("Organism|SA proportionality factor", sim)
param2 <- getParameter("Organism|Specific clearance (endosome)", sim)
param2$value <- NaN
param3 <- getParameter("Organism|Surface area scaling exponent", sim)

exportParametersToXLS(c(param1, param2, param3), paramsXLSpath = "../test.xlsx")
```

Reply:
Thanks, done in 9e4b492